datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/WeaklyTypedAspectsResolver.java

```diff
@@ -0,0 +1,84 @@
+package com.linkedin.datahub.graphql;
+
+import com.linkedin.data.DataMap;
+
+import com.linkedin.data.codec.JacksonDataCodec;
+import com.linkedin.datahub.graphql.generated.AspectParams;
+import com.linkedin.datahub.graphql.generated.AspectRenderSpec;
+import com.linkedin.datahub.graphql.generated.Entity;
+import com.linkedin.datahub.graphql.generated.EntityType;
+import com.linkedin.datahub.graphql.generated.RawAspect;
+import com.linkedin.datahub.graphql.resolvers.EntityTypeMapper;
+import com.linkedin.entity.client.EntityClient;
+import com.linkedin.metadata.models.AspectSpec;
+import com.linkedin.metadata.models.EntitySpec;
+import com.linkedin.metadata.models.registry.EntityRegistry;
+import com.linkedin.r2.RemoteInvocationException;
+import graphql.schema.DataFetcher;
+import graphql.schema.DataFetchingEnvironment;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import static com.linkedin.datahub.graphql.resolvers.ResolverUtils.*;
+
+
+@Slf4j
+@AllArgsConstructor
+public class WeaklyTypedAspectsResolver implements DataFetcher<CompletableFuture<List<RawAspect>>> {
+
+    EntityClient _aspectClient;
+    EntityRegistry _entityRegistry;
+    private static final JacksonDataCodec CODEC = new JacksonDataCodec();
+
+    private boolean shouldReturnAspect(AspectSpec aspectSpec, AspectParams params) {
+      return !params.getAutoRender() || aspectSpec.isAutoRender();
```

Comment:
This is confusing on first read -- 

I only return an aspect if there is no "auto render" params OR if the aspect is auto-render enabled? (Not clear what the difference is, yet)